docs/source/portfolio_optimisation/risk_estimators.rst

```diff
@@ -396,6 +415,41 @@ Implementation
    :members: denoise_covariance
 
 
+----
+
+Hierarchical Cluster Filtering  
+##############################
+
+Hierarchical Clustering, unlike K-means Clustering, does not create multiple clusters of identical size, nor does it require a pre-defined number of clusters. Of the two different types of heirarchical clustering - Agglomerative and Divisive - Agglomerative, or bottom-up clustering is used here. 
+
+Agglomerative Clustering assigns each observation to it's own individual cluster before iteratively joining the two most similar clusters. This process repeats until only a singluar cluster remains.
+
+Given a positive emperical correlation matrix, :math:`C` generated using :math:`n` features, the procedure given below returns as an output a rooted tree and a filtered correlation maxtrix :math:`C^<` of elements :math:`c^<_{ij}`. 
+
+First, set :math:`C = C^<`. 
+
+Then, beginning with the most highly correlated features (clusters) :math:`h` and :math:`k \in C` and the correlation between them, :math:`c_{hk}`, one sets the elements :math:`c^<_{ij} = c^<_{ji} = c_{hk}`. 
+
+The matrix :math:`C^<` is then redefined such that:
+
+.. math::
+    \begin{cases} c^<_{qj} = f(c^<_{hj}, c^<_{kj}) & where \ j \notin h \ and \ j \notin k \\ c^<_{ij} = c^<_{ij} & otherwise \end{cases}
+
+where :math:`f(c^<_{hj}, c^<_{kj}` is any distance metric. In effect, merging the clusters :math:`h` and :math:`k`. These steps are then completed for the next two most similar clusters, and are repeated for a total of :math:`n-1` iterations; until only a single cluster remains.
+
+
+.. tip::
+    Divisive Hierarchical clustering works in the opposite way. It starts with one single cluster wrapping all datapoints and divides the cluster at each step of it's iteration until it ends with n clusters.
```

Comment:
Link to the original paper here or in the tip at the beginning of the page.

`it's` to `its`